google-cloud-core/src/main/java/com/google/cloud/HttpTransportOptions.java

```diff
@@ -154,10 +155,26 @@ public void initialize(HttpRequest httpRequest) throws IOException {
         if (readTimeout >= 0) {
           httpRequest.setReadTimeout(readTimeout);
         }
+
+        HttpHeaders headers = httpRequest.getHeaders();
+        headers.set("x-goog-api-client", getXGoogApiClientHeader());
       }
     };
   }
 
+  String getXGoogApiClientHeader() {
+    return String.format(
+        "gl-java/%s %s/%s",
```

Comment:
The current lack of `gapic` or `grpc` in the header, and then the presence of one or both later, tells us this story.

You can feel free to add `gdcl` if you like. There's just no plan to track it unless it's the top-most client library in the stack (i.e. `gccl` is not present, since `gccl` always layers on `gdcl`).